testing/unit-tests/class.Text_Diff_Renderer_parallel.php

```diff
@@ -12,7 +12,7 @@ class Text_Diff_Renderer_parallel extends Text_Diff_Renderer
 	var $final    = 'Final';
 	var $_leading_context_lines = 10000; // these are big to ensure entire string is output
 	var $_trailing_context_lines = 10000;
-	function _blockHeader() {}
+	function _blockHeader($xbeg, $xlen, $ybeg, $ylen) {}
```

Comment:
This was just to fix a mismatched signature error that PHP died on when trying to run the tests. I tried installing the `RELEASE_1_1_1` tag of the Text_Diff lib (to match the version mentioned in the comment at the top of the tests file), but that one has the signature with the params as well: https://github.com/pear/Text_Diff/blob/RELEASE_1_1_1/Diff/Renderer.php#L183